modules/ROOT/pages/tokenization-example.adoc

```diff
@@ -4,49 +4,48 @@
 
 To configure the tokenization service, you need the following:
 
-* A Runtime Fabric 1.1.153 or later with an ingress configured. See xref:1.0@runtime-fabric::index.adoc[the Runtime Fabric documentation].
-* A secrets group to store the tokenization table encryption keys. The tokenization service creates and stores its own keys. See xref:asm-secret-group-concept.adoc[Secrets Manager documentation].
+* Runtime Fabric 1.1.153 or later with an ingress configured. See xref:1.0@runtime-fabric::index.adoc[the Runtime Fabric documentation].
+* A secret group to store the tokenization table encryption keys. The tokenization service creates and stores its own keys. See xref:asm-secret-group-concept.adoc[Secrets Manager documentation].
 * A tokenization format, which describes the format and how data is tokenized. See xref:tokenization-formats.adoc[Tokenization Formats].
 * A tokenization service that handles the actual tokenization requests and also tokenizes and detokenizes the data. For security purposes, an API Gateway must protect this service.
 * API Gateway, which proxies requests to the tokenization service, provides authorization and authentication, and provides JSON protection for the tokenization service.
 
-The reference model for this example tokenization service setup can be seen in the below diagram. In this diagram, a “Sensitive Data - Compliance Zone” is used to segregate the tokenization service from the general Runtime Fabric. This segregation is based on security needs.
 
-For example, if sensitive data is produced from a Mule Application serving external traffic (Client -> Edge -> Mule Application), this application might be included in the sensitive data zone.
+image::tokenization-setup-example-diagram.png[Tokenization Examples]
 
-image::tokenization-setup-example-diagram.png[]
+In this diagram, a “Sensitive Data - Compliance Zone” is used to segregate the tokenization service from the general Runtime Fabric. This segregation is based on security needs.
```

Comment:
if you want to keep the caps & quotation marks, change "a" to "the"? But since the diagram is clear, I'd just say "In this diagram, a separate zone segregates the tokenization..."